alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/MonitorRunner.kt

```diff
@@ -248,22 +250,29 @@ class MonitorRunner(
                     if (it.value.throttled) 1 else 0) })
         }
 
+        if (alertSchemaVersion == null) {
```

Comment:
I don't see any other places that uses `alertSchemaVersion` out side of this function and this if-statement will always be true because of that. Can we keep `alertSchemaVersion` scope within the function and this can also help to address lucaswin's comment about using `!!`

Reply:
If move `alertSchemaVersion` within the function, have to get `alertSchemaVersion` for every `composeAlert` call. So prefer to keep it as class property.

Reply:
After reading the PR again, we can modify SchemaVersionUtils a bit to make it more immutable.

`SchemaVersionUtils` should have functions to get schemaversion to get specific index version. ie. `SchemaVersionUtils.getAlertIndexSchemaVersion()` and `SchemaVersionUtils.getScheduledJobSchemaVersion()`. In those functions, we can hide this logic of initializing once if such schema version does not exist. This way if in future other class needs `AlertIndexSchemaVersion` or `ScheduledJobSchemaVersion` that class does not need to build this complicated logic.

Reply:
SchemaVersionUtils is in `core` package now, so it can't access alerting index as it's in `alerting` package. 
Will add extra util class in alerting package and initialize alert and scheduled job index schema version set